lib/Frontend/ModuleInterfaceLoader.cpp

```diff
@@ -736,13 +737,7 @@ class ModuleInterfaceLoaderImpl {
     // Should we attempt to load a swiftmodule adjacent to the swiftinterface?
     bool shouldLoadAdjacentModule = !ctx.IgnoreAdjacentModules;
 
-    // Don't use the adjacent swiftmodule for frameworks from the public
```

Comment:
I personally think this comment is still useful.

Reply:
Added back!